cmd/envvarlinter/README.md

```diff
@@ -0,0 +1,31 @@
+# envvarlinter
+
+envvarlinter ensures that non-test files don't use os.Getenv and os.LookupEnv and instead use the functions from pkg/env.
+
+envvarlinter is based on [Checker](../README.md), and this package provides the [custom rules](rules) implementation.
```

Comment:
broken link?

Reply:
Indeed. I'm surprised the markdown linter didn't catch this. Anyway, I removed the whole line.